src/sass/mixins.scss

```diff
@@ -83,27 +83,6 @@
   }
 }
 
-@mixin show-tooltip-on-hover {
```

Comment:
Why can we remove all of this?

Reply:
Same for other tooltip css

Reply:
When searching I didn't find a place where the style was applied so I classified as dead code and removed as well.